control/control.go

```diff
@@ -127,6 +127,8 @@ type managesPlugins interface {
 	SetMetricCatalog(catalogsMetrics)
 	GenerateArgs(logLevel int) plugin.Arg
 	SetPluginConfig(*pluginConfig)
+	SetPluginTags(map[string]map[string]string)
+	AddStandardAndWorkflowTags(core.Metric, map[string]map[string]string) core.Metric
```

Comment:
I don't understand "test won't pass" as an explanation. The test were passing before without that method on the interface?